src/test/scala/nl.knaw.dans.easy.multideposit/BlackBoxSpec.scala

```diff
@@ -61,6 +53,10 @@ class BlackBoxSpec extends UnitSpec with MockFactory with CustomMatchers {
   def beforeAll(): Unit = {
     Paths.get(getClass.getResource("/allfields/input").toURI).copyDir(allfields)
     Paths.get(getClass.getResource("/invalidCSV/input").toURI).copyDir(invalidCSV)
+
+    it should "contain certain Formats" in {
+      formats.contains("audio/mpeg3") shouldBe true
```

Comment:
I'm not sure whether this test is actually executed inside the `it should ...` context. Never tested that. I usually would write this test without the `it should ...` ceremony.